install/gcp/installer/forseti_server_installer.py

```diff
@@ -77,6 +77,14 @@ def deploy(self, deployment_tpl_path, conf_file_path, bucket_name):
             bool: Whether or not the deployment was successful.
             str: Deployment name.
         """
+        self.has_roles_script = gcloud.grant_server_svc_acct_roles(
+            self.enable_write_access,
+            self.access_target,
+            self.target_id,
+            self.project_id,
+            self.gcp_service_acct_email,
+            self.user_can_grant_roles)
+
```

Comment:
Will you still be adding the time out here?  I think we still need it to mitigate the race condition.  With the previous comment would be awesome.

Reply:
Added in.